src/App.tsx

```diff
@@ -4,7 +4,7 @@ const GlobalStyle = createGlobalStyle`
   font-family: Open Sans, Montserrat, Roboto
 `;
 
-export default function App() {
+export default function App(): JSX.Element {
```

Comment:
I personally like explicitly declaring the functional component's type for a number of reasons.

1. It's clear what will you return with the component, sometimes you simply return primitive types such as `string`, `number` etc.
2. When the component has logic to hide itself, you'll need to declare it as `JSX.Element | null` which helps other devs to realize there's a logic to hide the component declaratively